src/test/java/io/javalin/TestEncoding.java

```diff
@@ -18,7 +18,7 @@
 
     @Test
     public void test_param_unicode() throws Exception {
-        app.get("/:param", ctx -> ctx.result(ctx.pathParam("param")));
+        app.get("/:pathParam", ctx -> ctx.result(ctx.pathParam("pathParam")));
```

Comment:
Use kebab case in URLs: `path-param`.

Reply:
ok, I changed it in other files as well.